app/controllers/message_drafts_controller.rb

```diff
@@ -32,9 +32,12 @@ def destroy
     # redirect_path = @message.original_message.present? ? message_thread_path(@message.original_message.thread) : message_drafts_path
     redirect_path = @message.original_message.present? ? message_thread_path(@message.original_message.thread) : message_threads_path
 
-    @message.destroy
-
-    redirect_to redirect_path, notice: "Správa bola zahodená"
+    if @message.not_yet_submitted?
```

Comment:
@luciajanikova vies to pozriet ci je toto dobry scope?

Reply:
Ano, toto je ok